python/dgl/distributed/graph_partition_book.py

```diff
@@ -2,14 +2,13 @@
 
 import pickle
```

Comment:
There are multiple classes defined in this file, could be better if we can break it to multiple pyfiles.

Reply:
they are put together probably because they are quite related, I think. Base class, derived classes, highly related classes. Maybe we could re-locate them in a separate PR to be clear, if you insist on this.